expression/builtin_math.go

```diff
@@ -393,6 +393,7 @@ func (c *ceilFunctionClass) getFunction(args []Expression, ctx context.Context)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	fixFloorAndCeilReturnFlag(&bf, args[0])
```

Comment:
can we change the name `fixFloorAndCeilReturnFlag` and `fixFloorAndCeilType` to a more meaningful one ?